src/sparse/impl/KokkosSparse_spiluk_symbolic_impl.hpp

```diff
@@ -358,6 +342,18 @@ void iluk_symbolic ( IlukHandle& thandle,
   thandle.set_nnzL(cntL);
   thandle.set_nnzU(cntU);
 
+  // Sort
+  for (size_type row_id = 0; row_id < static_cast<size_type>(L_row_map.extent(0))-1; row_id++) {
+    size_type row_start = L_row_map(row_id);
+    size_type row_end   = L_row_map(row_id + 1);
+    Kokkos::sort(subview(L_entries, Kokkos::make_pair(row_start, row_end)));
+  }
```

Comment:
@vqd8a For this, you could do:
```
#include "KokkosKernels_Sorting.hpp"
...
KokkosKernels::sort_crs_graph<exec_space, decltype(L_row_map_d), decltype(L_entries_d)>
(L_row_map_d, L_entries_d);
```
Then it would sort the rows in parallel on device.

Reply:
@brian-kelley Thanks. This is useful. I can try it. Is ```KokkosKernels::sort_crs_graph``` in ```KokkosKernels_SparseUtils.hpp``` or in ```KokkosKernels_Sorting.hpp```? 
It looks to me in Kokkos Kernels's ```develop``` branch, it is in ```KokkosKernels_Sorting.hpp``` but in Trilinos, it is in ```KokkosKernels_SparseUtils.hpp```.

Reply:
@vqd8a I moved all the sorting stuff into KokkosKernels_Sorting.hpp just before the 3.4 release. So if you're in Trilinos, you would do ``#include "KokkosKernels_SparseUtils.hpp" ... KokkosKernels::Impl::sort_crs_graph...``, and if you're in KokkosKernels develop, do ``#include "KokkosKernels_Sorting.hpp" ... KokkosKernels::sort_crs_graph...``.

Reply:
@brian-kelley in this case, how can I match this Kokkos Kernels PR with trilinos/Trilinos#9088? Or does trilinos/Trilinos#9088 have to wait for the next release of Kokkos Kernels to be in Trilinos? 

Another thing is I tried 
```
#include "KokkosKernels_SparseUtils.hpp"
KokkosKernels::Impl::sort_crs_graph...
```
in Trilinos but I got compile error  ```error: static assertion failed with "View allocation constructor requires managed memory"```
It looks to me that ```sort_crs_graph``` does  not work with unmanaged views (e.g. ```L_row_map_d, L_entries_d```). Do you have any idea?

Reply:
@vqd8a Oh, I know what's going on. sort_crs_graph sometimes allocates temporary buffers with the same type as the Entries, and yours are unmanaged. But that case should be supported. I'll fix it. If you want to get this PR merged and then patched into Trilinos quickly, then you should just keep the serial host sorting for now.

Reply:
@brian-kelley Will #960 be in Trilinos soon?

Reply:
@vqd8a Probably in the minor release 3.4.1. I don't know when that will be but it'll be much quicker than a normal promotion.

Reply:
@brian-kelley Thanks. I think for now I will just keep the serial host sorting in this PR. But I will keep this in my mind and will change it to use ```sort_crs_graph``` in another PR.